runtime/gc_base/modronapi.cpp

```diff
@@ -1041,18 +1041,43 @@ continuationObjectCreated(J9VMThread *vmThread, j9object_t object)
 	Assert_MM_true(NULL != object);
 	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
 
-	if (MM_GCExtensions::disable_continuation_list != MM_GCExtensions::getExtensions(env)->continuationListOption) {
+	if (MM_GCExtensions::onCreated == MM_GCExtensions::getExtensions(env)->timingAddContinuationInList) {
+		addContinuationObjectInList(env, object);
+	}
+	MM_ObjectAllocationInterface *objectAllocation = env->_objectAllocationInterface;
 
-		env->getGCEnvironment()->_continuationObjectBuffer->add(env, object);
-		MM_ObjectAllocationInterface *objectAllocation = env->_objectAllocationInterface;
+	if (NULL != objectAllocation) {
+		objectAllocation->getAllocationStats()->_continuationObjectCount += 1;
+	}
+	return 0;
+}
 
-		if (NULL != objectAllocation) {
-			objectAllocation->getAllocationStats()->_continuationObjectCount += 1;
-		}
+UDATA
+continuationObjectStarted(J9VMThread *vmThread, j9object_t object)
+{
+	Assert_MM_true(NULL != object);
+	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread);
+	if (MM_GCExtensions::onFirstMount == MM_GCExtensions::getExtensions(env)->timingAddContinuationInList) {
```

Comment:
let's rename this enum to onStarted to be consistent with the rest of related APIs/states